data/test_yaml_stories/test_prediction_with_correct_intent_wrong_entity.yml

```diff
@@ -0,0 +1,6 @@
+stories:
+- story: test story with correct intent and wrong entity
```

Comment:
Something else that I tested locally with the normal formbot but did not add to the tests because I did not want to make it slower (I would have to add another entity ?), is to have more than one entities. For example, have one wrongly predicted entity and one correctly predicted entity. That way I can check if it also prints all the entities regardless if there is a wrong prediction or not.

Reply:
I think it would make sense to have this in the test.

Reply:
I added it.